src/Debugger.php

```diff
@@ -39,6 +39,8 @@ public function __construct(Storage $storage, Event $event)
         $this->storage = $storage;
         $this->event = $event;
 
+        $this->responseKey = config('api-debugger.response_key', 'debug');
```

Comment:
Send this as a dependency through constructor, please. Or make a setter and overwrite in `boot()` method in ServiceProvider

Reply:
I've added to `ServiceProvider::boot()`. Although I was not able to create a test for this in `DebuggerTest` because `ServiceProvider::boot()` has ran in the `setUp()` before the tests run. Any ideas how to test this feature?

Reply:
There is no need to test service provider. Just add a test for `setResponseKey` and that will be enough.